### PR TITLE
swap S3.1 and 3.2; add S3 intro

### LINKED ISSUE #96: Switch the order of Sections 3.1 and 3.2
The details of user weighting shouldn't come before we've described the interface or defined what counts as a classification.

(Feel free to assign me!)

### PR DESCRIPTION
**Pull request by vrooje**
closes #96; 
also addresses one of the points on Carie's list in #109

### CODE REVIEW

File: writeup/gz_hubble_data.tex [hunk 1]
```diff
@@ -1,4 +1,4 @@
-\documentclass[twocolumn]{aastex}
+\documentclass[twocolumn]{aastex6}
```
Comment (willettk) [inline writeup/gz_hubble_data.tex:1]: Odd. Have you tried modifying the aastex6.cls file to change `aastex` -> `aastex6`?

**Comment by willettk** [comment]
Should be fine; I think the only thing for multiple in-progress edits might be to not commit the PDF. That's what git is having a harder time merging.

File: writeup/gz_hubble_data.tex (hunk 1 above)
Comment (vrooje) [inline writeup/gz_hubble_data.tex:1]: I believe I did that and it had some other weird issues... but maybe I was just afraid to. Will try.

**Comment by willettk** [comment]
Git had a harder time tracking the moving of the sections around, apparently. That, or maybe the smaller commits I did this afternoon.

File: writeup/gz_hubble_data.tex (hunk 1 above)
Comment (vrooje) [inline writeup/gz_hubble_data.tex:1]: The global `aastex` --> `aastex6` seemed to do the trick.

**Pull request merged**